### PR TITLE
First milestone - TIC-TAC-TOE

### PR DESCRIPTION
**Pull request by hillarioh**
Hi, 
This is the first milestone for the tic-tac-toe ruby project. It entailed creating of lib and bin directories. Inside the lib we added .gitkeep and made the main.rb executable. We have used the first-milestone branch.

### CODE REVIEW

File: bin/main [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env ruby
+puts 'Hello world!'
```
Comment (majovanilla) [inline bin/main]: * [ ] Your file should be executable using `bin/main.rb` as well as `ruby bin/main.rb` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/execute_bin.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

Right now, typing `bin/main.rb` returns a `permission denied` error.
Check this [link](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/) and make sure you followed all steps.

Also, I would advise you against removing the `.rb` extension of your file. It's not a good practice.

**Review by majovanilla — CHANGES_REQUESTED** [review]
Hi team!

## Good job on this project! 👍 You are almost done! 

## Please, make all REQUIRED changes below and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

File: bin/main (hunk 1 above)
Comment (hillarioh) [reply bin/main]: Hi majovanilla, 
Thank you for your review, I have renamed the files, the configuration for permissions are done on the local machine thus for that to run you will have to configure on your local machine.

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi team, 👋 

### Great job implementing the suggested changes. 🙌 Just one more thing to fix. 👇 

- [ ] Set up linter for your repo. Check [here](https://github.com/microverseinc/linters-config/tree/master/ruby).

**Kindly submit for another code review request once this has been done.** _Happy coding!_ 🎉

**Review by Torres-ssf — APPROVED** [review]
### PR Appproved :+1: :tada: 
Great job :+1: 
The linter was properly set up.
Good luck with the next PR.

Happy coding :computer: 

**Sergio Torres**

**Pull request merged**